### ISSUE TITLE
nullptr dereference in hb_ot_layout_get_baseline

### ISSUE THREAD

**Issue by kmilos** [harfbuzz/harfbuzz#3519]
This is w/ Windows (MSYS2) packages for both 4.0.1 and 4.1.0.

See https://github.com/msys2/MINGW-packages/issues/11124#issuecomment-1082002036

**Comment by kmilos** [comment]
Indeed, for some reason the font passed by Pango really seems to be null on Windows w/ default darktable theme: https://github.com/darktable-org/darktable/issues/10882#issuecomment-1021542865

**Comment by behdad** [comment]
cc @matthiasclasen 

We don't expect nullptr in our API.

**Comment by kmilos** [comment]
Done: https://gitlab.gnome.org/GNOME/pango/-/issues/680
